ccmlib/common.py

```diff
@@ -748,12 +748,12 @@ def pid_listening_on(addr: str, port: int):
                                  f'src = inet:{addr}:{port}'],
                                 stdout=subprocess.PIPE,
                                 check=True)
-        matched = re.search(r'pid=(\d+)', result.stdout)
+        matched = re.search(r'pid=(\d+)', str(result.stdout))
```

Comment:
probably we can instead pass `text=True` to `subprocess.run()`, so it can decode the output using the default encoding for us? like
```py
        result = subprocess.run([ss,
                                 '--no-header',
                                 '--listening',
                                 '--processes',
                                 '--tcp',
                                 f'src = inet:{addr}:{port}'],
                                stdout=subprocess.PIPE,
                                check=True,
                                text=True)
```

Reply:
added the `text=True`